BUILD.gn

```diff
@@ -613,6 +613,14 @@ executable("antares") {
     "//ext/glfw:libglfw",
   ]
   configs += [ ":antares_private" ]
+  defines = []
+  if (mode == "opt" && host_os == "win") {
+    configs -= [ "//build/lib/win:console" ]
+
+    configs += [ "//build/lib/win:windows" ]
+    defines += [ "WIN32_NO_CONSOLE" ]
```

Comment:
Actually, is this necessary now? Outside of opt mode, the win32 source file isn’t compiled.